src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -273,6 +278,15 @@ private string GetSize()
             return "1";
         }
 
+        private string escape(string content)
+        {
+            return content.Replace("\n", "\\n")
+                .Replace("\"", "&quot;")
+                .Replace("<", "&lt;")
+                .Replace(">", "&gt;")
+                .Replace("'", "&#39;");
```

Comment:
maybe I canuse JsonUtility to serialize for that I can't get the tokenTree object ,I can't  redefine the render interface.

---
In reply to: [75990249](https://github.com/dotnet/docfx/pull/631#discussion_r75990249) [](ancestors = 75990249,75989168,75988652)